src/test.js

```diff
@@ -64,6 +64,10 @@ export default function Test( settings ) {
 		this.async = false;
 		this.expected = 0;
 	} else {
+		this.module.unskippedTests.push( {
```

Comment:
We should create a single object up on line 55 and push it into both the `tests` and `unskippedTests` array, since we want that info to be in sync.

Also, I'd appreciate a flag `skipped` to be added to the object, for more info.